packages/trackerless-network/protos/NetworkRpc.proto

```diff
@@ -24,6 +24,10 @@ service NeighborUpdateRpc {
   rpc neighborUpdate (NeighborUpdate) returns (NeighborUpdate);
 }
 
+service InspectionService {
+  rpc inspectConnection (InspectConnectionRequest) returns (InspectConnectionResponse);
```

Comment:
It is used to open a connection to the inspected peer. When the server receives the connection it will start to propagate messages to the inspecting peer. In the simplified inspection it was decided that the inspection connections will have their own operational pipelines to not interfere the with the graph formation algorithm for a short lived functionality.

Reply:
Ok then the method could be something like `open*Connection`. E.g. openDirectConnection or something that describes the attributes and behavior of the connection. (The inspection word tells for what the connection is used at the moment, but it would be more descriptive to know how that connection differs from other connections). 

Also the inspection is a higher level concept, so maybe there is no need to know about that on protocol level.

Reply:
I see this as just a minor naming issue, but I'm pitching in as I was pinged for ideas. How about calling it a `temporary` or `adhoc` connection? Maybe one of those captures the correct semantic? 

Teo's point is valid in that the connection shouldn't care what it's being used for, as long as the other end knows how to handle the connection.

Temporary connectivity could have other uses in the future, beyond inspection?